exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/stream/QueryRunner.java

```diff
@@ -48,7 +48,7 @@ public void start(QueryWrapper query) throws ValidationException {
     startQuery(QueryType.valueOf(query.getQueryType()),
         query.getQuery(),
         userConn);
-    userConn.onStart(queryId, maxRows);
+    userConn.onStart(queryId, maxRows, options);
```

Comment:
Not needed. The connection already has this member:

```
  protected WebSessionResources webSessionResources;
```

Which provides:

```
  private final UserSession webUserSession;
```

Which provides `getOptions()`

Reply:
Thanks, used this method